### PR TITLE
Close #335 Add LE whitelist functions

### LINKED ISSUE #335: Add wrappers for whitelist functions
Currently, there is no support for hci_le_add_white_list, hci_le_read_white_list_size, and hci_le_clear_white_list. btmodule.c should be extended so as to wrap these functions.

### PR DESCRIPTION
**Pull request by alessandro-allen**
Extended module to include functions for adding LE address to the scan whitelist, read the size of the whitelist, and clear the whitelist.

### CODE REVIEW

File: bluez/btmodule.c [hunk 1]
```diff
@@ -2552,6 +2552,85 @@ Returns a new HCI filter suitable for operating on with the hci_filter_*\n\
...
+	err = hci_le_add_white_list(dd, &ba, type, to);
+	if ( err < 0 ) {
+		return NULL;
+	}
+
+	return Py_BuildValue("i", err);
+}
+
+PyDoc_STRVAR(bt_hci_le_add_white_list_doc,
+"hci_le_add_white_list(dd, btaddr, type, to)\n\
```
Comment (rgov) [inline bluez/btmodule.c]: The formatting here seems weird, is this what you intended?

File: bluez/btmodule.c [hunk 2]
```diff
@@ -2552,6 +2552,85 @@ Returns a new HCI filter suitable for operating on with the hci_filter_*\n\
...
+\n\
+Add the given MAC to the LE scan white list");
+
+static PyObject *
+bt_hci_le_read_white_list_size(PyObject *self, PyObject *args)
+{
+	PySocketSockObject *socko = NULL;
+        int err;
```
Comment (rgov) [inline bluez/btmodule.c]: Please fix indentation to be consistent.

**Review by rgov — CHANGES_REQUESTED** [review]
Thank you for adding this feature. Please see a couple formatting points above.

I think it's good to clean up whitespace issues in the code but future please do it in a separate commit (or even a separate pull request) next time so that it's easier to see which changes are unrelated formatting changes.

**Comment by rgov** [comment]
This is currently failing to build on our CI system.

File: bluez/btmodule.c (hunk 1 above)
Comment (alessandro-allen) [reply bluez/btmodule.c]: I tried to follow the format of the existing doc strings but (assuming my subsequent commit doesn't break the build) please let me know if you think this was incorrect.

**Comment by rgov** [comment]
Thank you!

**Pull request merged**